### ISSUE TITLE
Show soft-logged-out users in login screen

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#22032]
### Your use case

If i understand soft logout correctly, when an access token becomes invalid (or likewise), users will be kicked back to the login screen, but their user data (plus crucial encryption data) will remain.

Currently, there is no information that displays this on the login screen, it'd be useful to show a soft-logged-out account at the entry screens;
![image](https://user-images.githubusercontent.com/22740616/166459463-59106717-c775-4e04-8b26-afbd7cc597c2.png)
![image](https://user-images.githubusercontent.com/22740616/166459485-b933dc07-0676-4617-a0fa-ce9d159eb154.png)

It'd show some basic info (MXID, Display Name, Avatar, "Signed Out"), and when clicking it, it will present the user with a UIA flow (password etc.) to log back in.

It'd also help if there was an indicator of "Encryption Information Stored" or the likes, that the user is aware that encryption information is cached, but that it has been signed out of the homeserver session, and that logging back in will preserve this encryption information.

### Additional context

My inspiration comes from what google does, it has a "marker" of a user that was logged in, but displays "signed-out" to show that the user would need to log back in to access this account;

![Uploading image.png…]()

**Comment by t3chguy** [comment]
> If i understand soft logout correctly, when an access token becomes invalid (or likewise), users will be kicked back to the login screen, but their user data (plus crucial encryption data) will remain.

Yes but no, the server has to specify it is a soft logout, by default it is a hard logout (e.g de-authing from your own device because your laptop was stolen)

This is what it looks like

![image](https://user-images.githubusercontent.com/2403652/166460378-0c955209-38c1-4709-b859-aee053060997.png)

but yes still lacks the metadata you asked for

> It'd show some basic info (MXID, Display Name, Avatar, "Signed Out"), and when clicking it, it will present the user with a UIA flow (password etc.) to log back in.

/login doesn't support UIA

**Comment by ShadowJonathan** [comment]
Is that screenshot current? Because that's pretty much exactly what I'm looking for.

If this is already live, then this issue can be closed.

**Comment by t3chguy** [comment]
> If this is already live, then this issue can be closed.

I think being asked to input a password without the context of what account for is crap, so I disagree

**Comment by ShadowJonathan** [comment]
I thought the same, but I considered it an upgrade over nothing. I'll keep this open, then.
